cookbooks/stac-zarr-report/data-consumers/index.qmd

```diff
@@ -0,0 +1,90 @@
+
+### Filter in STAC first
+
+If you are interacting with a STAC API and the Zarr store is cataloged in STAC following `xstac` conventions then all the metadata you need for filtering and subsetting is available at the STAC level so you should be able to search either in the web-browser directly or using pystac-client.
+
+If you are interacting with a static STAC catalog (in json or geoparquet) you might be able to use stac-rs and stac-geopandas to search the collection and find the items of interest. _This is a newer approach and might have rough edges_.
+
+In both the API and static STAC case the result of the search will be an iterable of items. Each of these items will have at least one asset pointing to the Zarr store (sometimes there will be multiple assets that point to the same store using different protocols). Take the assets that meet your needs and you can use `xr.open_mfdataset` with the `zarr` or `stac` backend to construct the data cube.
+
+If the Zarr stores all cover the same area and are on the same grid then concatenation might be straightforward. If you need to apply some transformation (such as regridding or doing a projection) before concatenating the data in the files then you might find it easier to use  `xr.open_dataset` directly and iterate over the items in the list applying transformations before using one of xarray's combine methods.
+
+
+## Virtual reference files in STAC
+
+Virtual reference files can be treated exactly like how you would treat a Zarr store. They can be lazily-loaded into xarray using the [`kerchunk`][5] or [`stac`][1] backend and doing so will result in one GET request containing the minimal metadata.
+
+The xarray `stac` backend has experimental support for reading virtual references stored directly in the STAC items (no external file).
+
+
+## Storing results
+
+This is a newer area of development. It applies to cases where you are interacting with many smaller Zarr stores and concatenating them. The core concept is that instead of repeatedly querying the STAC catalog you could store the results for easy access.
+
+There are several layers at which you can store results:
+  - Store the result of a STAC search in [stac-geoparquet][2]  (using [stac-rs][3])
+  - Store the result of a data cube constructed by concatenating Zarr stores: 
+    - as a new Zarr store - this option can include filtering and subsetting
+    - as a virtual reference file ([icechunk][6] or [kerchunk][5])
+
+## Future Work
+
+- It might be possible to support filtering and subsetting inside one big Zarr by adding predicate push-down to [xpystac][1]
+
+## Links
+
+- [xpystac][1]
+- [stac-geoparquet][2]
+- [stac-rs][3]
+- [kerchunk][5]
+- [icechunk][6]
+- [Earthmover blog: Accelerating Xarray with Zarr-Python 3][4]
+- [Collection Search API Extension][7]
+
+[1]: https://github.com/stac-utils/xpystac
+[2]: https://github.com/stac-utils/stac-geoparquet
+[3]: https://github.com/stac-utils/stac-rs
```

Comment:
Since we're in Python land, I think we should just point to the Python package repo (not the Rust one):

```suggestion
[3]: https://github.com/stac-utils/rustac-py
```